docker-compose.yml

```diff
@@ -63,7 +63,7 @@ services:
       - MHS_SDS_REDIS_CACHE_HOST=redis
       - MHS_SDS_REDIS_DISABLE_TLS=True
   dynamodb:
-    image: nhsdev/nia-dynamodb-local:1.0.1
+    image: nhsdev/nia-dynamodb-local:1.0.2
```

Comment:
I'm not sure why the version got bumped here. Have we or when will we release the new version to Dockerhub?

Reply:
It should be discussed, I think. We have not already release it.

Reply:
There is a release.sh script in the /dockers/common/dynamodb-local folder that you use to build/tag/push the image. However you need to be logged into the docker hub as a user that has access to nhsdev. I have it, will do this for you now.

Reply:
Thank you for your help, @petervdm